German Translation

DahliaSR opened the pull request:
Closes #1

DahliaSR committed:
Translated Short Description

Part of #1

DahliaSR committed:
Fixed Typo

Part of #1

DahliaSR committed:
Startet Translation of Description

Part of #1

DahliaSR committed:
Finished Translation of Description

Part of #1

DahliaSR commented:
@GiForce 
Your approval is pending...

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -129,41 +135,43 @@
-    This task shows the variance of the positional ciphers.
-    With this cipher you can hide a message within a book which allows you and receiver to send these coded messages.
+    <strong>Wozu das gut ist: </strong>
+    Diese Aufgabe zeigt dir eine Variante der <a href="https://de.wikipedia.org/wiki/Buch-Verschl%C3%BCsselung">Buch- oder auch Ottendorf-Verschlüsselung</a>.
```
...zeigt **D**ir...

DahliaSR replied:
:white_check_mark:

DahliaSR committed:
Fixes according to review

https://github.com/CheckIO-to-German/The-Hidden-Word/pull/2#pullrequestreview-4325500

GiForce reviewed (approved):
🎃
